### ISSUE TITLE
new format, new finetune code?

### ISSUE THREAD

**Issue by ehartford** [nlpxucan/WizardLM#64]
Llamax code it knows how to handle alpaca formatted QA code, but I didnt' see anything in there to handle ShareGPT format code,

How do I finetune with the new format?  Your finetune guide (https://github.com/nlpxucan/WizardLM/blob/main/WizardLM/README.md#fine-tuning) still references the 70k dataset

**Comment by chat-guy** [comment]
>  Your finetune guide (https://github.com/nlpxucan/WizardLM/blob/main/WizardLM/README.md#fine-tuning) still references the 70k dataset

Yep, noticed this too. Perhaps @nlpxucan forgot to update that particular section of the readme during the last commit (2 days ago).

**Comment by ehartford** [comment]
perhaps this

https://github.com/nlpxucan/WizardLM/blob/94f9c792df4b91589c8c236a566ddc63d4868ec2/WizardLM/src/train_freeform.py#LL50C5-L50C17

**Comment by ehartford** [comment]
Or maybe they used FastChat rather than Llamax.

**Comment by ehartford** [comment]
gonna presume we are using FastChat until I hear otherwise.

**Comment by anmolagarwal999** [comment]
@nlpxucan Bumping this issue up for your attention. Thanks !
